kmath-core/build.gradle.kts

```diff
@@ -5,10 +5,21 @@ plugins {
 //    id("com.xcporter.metaview") version "0.0.5"
 }
 
-kotlin.sourceSets {
-    commonMain {
-        dependencies {
-            api(project(":kmath-memory"))
+kotlin {
+    jvm {
+        compilations.all {
+            kotlinOptions {
+                freeCompilerArgs =
+                    freeCompilerArgs + "-Xjvm-default=all" + "-Xopt-in=kotlin.RequiresOptIn" + "-Xlambdas=indy"
```

Comment:
Won't it break compatibility somehow?

Reply:
It doesn't change the API at all.

Reply:
The only compatibility change is the `toString` value of objects returned by `binaryOperaionFunction`.